### PR TITLE
Feature/k3v time

### PR DESCRIPTION
**Pull request by Dmitry-White**
**What was done**
Add a real world data fetching for time and weather conditions by given coordinates of the building.
Add an automatic class setting based on fetched data.
Add a simple time displaying with setTimeout updates of DOM each second.

### CODE REVIEW

File: src/js/app.js [hunk 1]
```diff
@@ -1,17 +1,91 @@
...
 import Building from './building';
 import Console from './console';
+import { getTime, getWeather } from './utils';
+import { clouds, condition } from './options.json';
 
 export default class App {
     constructor() {
-        this.sky = new Sky(300, 15);
-        this.building = new Building(20, 3);
         this.console = new Console();
```
Comment (elkinny) [inline src/js/app.js]: Why Sky and Building creation moved to "create" function, and the Console not?

File: src/js/app.js [hunk 2]
```diff
@@ -1,17 +1,91 @@
...
+        for (let key in condition) if (this.condition.includes(key)) this.mode.push(key);
+
+        this.create();
+    };
+
+    create() {
+        console.log("Temperature: ", this.temperature);
+        console.log("Mode: ", this.mode);
+
+        let cloudsCount = clouds[this.clouds];
```
Comment (elkinny) [inline src/js/app.js]: Dima, wtf :) salt is salty

File: src/js/sky/sky.js [hunk 3]
```diff
@@ -1,22 +1,23 @@
...
         this.cloudsCount = cloudsCount;
+        this.mode = mode;
     }
 
     fill() {
         const sky = document.createDocumentFragment();
         const cloud = document.createElement('div');
 
-        cloud.classList.add('cloud');
+        cloud.classList.add('cloud', ...this.mode);
```
Comment (elkinny) [inline src/js/sky/sky.js]: nice 👍

File: src/js/app.js (hunk 1 above)
Comment (Dmitry-White) [reply src/js/app.js]: Implementation concerns: these two components depend on fetched data.
Will be refactored in future.

File: src/js/sky/sky.js (hunk 3 above)
Comment (Dmitry-White) [reply src/js/sky/sky.js]: Oh stop it you 😊

File: src/js/app.js (hunk 2 above)
Comment (Dmitry-White) [reply src/js/app.js]: Well, duh 🙄

**Pull request merged**